test/integration/vitesscluster/vitesscluster_test.go

```diff
@@ -137,6 +173,41 @@ func TestBasicVitessCluster(t *testing.T) {
 	verifyBasicVitessCluster(f, ns, cluster)
 }
 
+func TestVtbackupExtraFlagsPropagated(t *testing.T) {
+	ctx := context.Background()
+
+	f := framework.NewFixture(ctx, t)
+	defer f.TearDown()
+
+	ns := "default"
+	cluster := "test-vtbackup-extra-flags"
+	keyspace := "ks"
+	shard := "-"
+
+	f.CreateVitessClusterYAML(ns, cluster, vtbackupExtraFlagsCluster)
+
+	// Wait for the vtbackup init pod to be created for the only shard.
+	podName := names.JoinWithConstraints(names.DefaultConstraints, cluster, keyspace, shard, "vtbackup", "init")
+	var pod corev1.Pod
+	f.MustGet(ns, podName, &pod)
+
+	// Find the vtbackup container and check args contain extra flags.
+	var args []string
+	for _, c := range pod.Spec.Containers {
+		if c.Name == "vtbackup" {
+			args = c.Args
+			break
+		}
+	}
+	joined := strings.Join(args, " ")
+	if !strings.Contains(joined, "--foo=bar") {
+		f.Fatalf("vtbackup args missing --foo=bar: %v", args)
+	}
+	if !strings.Contains(joined, "--baz=qux") {
+		f.Fatalf("vtbackup args missing --baz=qux: %v", args)
+	}
```

Comment:
```suggestion
	require.Contains(t, joined, "--foo=bar", "vtbackup args missing --foo=bar: %v", args)
	require.Contains(t, joined, "--baz=qux", "vtbackup args missing --baz=qux: %v", args)
```

Reply:
I blindly updated this, however, I'll note that "require" is currently only also used in `test/integration/framework/fixture.go`.  The `strings` form I used originally is used elsewhere.
Am I to assume this is just the direction you want to move, and thus the suggested change?